google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/BQTableSchemaToProtoDescriptor.java

```diff
@@ -1,7 +1,6 @@
 /*
  * Copyright 2020 Google LLC
- *
```

Comment:
The bot incorrectly proposed these changes. Raised [this issue](https://github.com/googleapis/code-suggester/issues/163) to track a fix.